sdks/python/apache_beam/io/hdfilesystem.py

```diff
@@ -0,0 +1,190 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""FileSystem implementation for accessing Hadoop Distributed File System
```

Comment:
<!--thread_id:cc_156236477_t; commit:3c708ac10af4457fe46e07503f0b72930832db07; resolved:1-->
<!--section:context-quote-->
> **chamikaramj** wrote:
> :class:\`~apache_beam.io.filesystem.FileSystem\`
> 
> Also please updated other class/method references in py docs to use either above or \`\`Foo\`\` notation so that they get properly formatted.

<!--section:body-->
Done.